src/mkdocstrings_handlers/python/handler.py

```diff
@@ -257,6 +259,7 @@ def collect(self, identifier: str, config: Mapping[str, Any]) -> CollectorItem:
                 docstring_options=parser_options,
                 modules_collection=self._modules_collection,
                 lines_collection=self._lines_collection,
+                allow_inspection=final_config["inspect_modules"],
```

Comment:
I think you meant this:

```suggestion
                allow_inspection=final_config["allow_inspection"],
```